index.js

```diff
@@ -1,17 +1,19 @@
 const core = require('@actions/core');
-const wait = require('./wait');
+const docsington = require('./src/main');
 
 
 // most @actions toolkit packages have async methods
 async function run() {
   try {
-    const ms = core.getInput('milliseconds');
-    core.info(`Waiting ${ms} milliseconds ...`);
+    // TODO this input needs to be updated to match docsington
```

Comment:
This is all fluff, right? As in, we don't actually need it. The only thing we need right now is calling docsington(words). It's fine, I'm not saying we should remove it - just confirming what's what.

Reply:
The `getInput` and `setOutput` stuff are needed. They act as the interface of our github action with the other values in the github workflow and we'll need one `getInput` for each of the values we pass to the action. Ie. see the example YML snippet below (from our slack convo):

```
    - name: Run Automated Document Audits
      uses: actions/docsington@v1.0.0
      with:
        owner: @ericmjalbert
```

We'll need to have a `core.getInput('owner');` to be able to use the `@ericmjalbert` value.